lab-aaron/lib/greet.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+let theUser = module.exports = {};
+
+theUser.hello = function(a){
+  if (a) {
```

Comment:
The technical requirements for this function included validating that the input was a string (not a number, Boolean, etc.)